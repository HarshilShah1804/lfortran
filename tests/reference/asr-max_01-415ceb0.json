{
    "basename": "asr-max_01-415ceb0",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/max_01.f90",
    "infile_hash": "3154045abbca74724d67eeb8e4744c66f702ec1a598db55701c43729",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-max_01-415ceb0.stdout",
    "stdout_hash": "eb9d7af4c76b60a99986df7442d82524e967f2de6fc02ad85dbfabbb",
    "stderr": "asr-max_01-415ceb0.stderr",
    "stderr_hash": "ffae0278fb457614873da2a97daf1953347eda5f0a0585016b6cf07c",
    "returncode": 0
}