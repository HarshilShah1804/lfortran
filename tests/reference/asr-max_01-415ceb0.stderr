warning: Different kinds of args in max0 is a non-standard extension
 --> tests/../integration_tests/max_01.f90:6:14
  |
6 |     print *, max(y, z)
  |              ^^^^^^^^^ help: ensure all arguments have the same kind to make it standard

warning: Different kinds of args in max0 is a non-standard extension
 --> tests/../integration_tests/max_01.f90:7:14
  |
7 |     if (abs (max(y, z) - 9.0d0) > 1d-16) error stop "Incorrect value of max"
  |              ^^^^^^^^^ help: ensure all arguments have the same kind to make it standard

warning: Different kinds of args in max0 is a non-standard extension
 --> tests/../integration_tests/max_01.f90:8:36
  |
8 |     print *, "Kind of max:", kind( max( y, z))
  |                                    ^^^^^^^^^^ help: ensure all arguments have the same kind to make it standard

warning: Different kinds of args in max0 is a non-standard extension
 --> tests/../integration_tests/max_01.f90:9:16
  |
9 |     if ( kind( max( y, z)) /= 8 ) error stop
  |                ^^^^^^^^^^ help: ensure all arguments have the same kind to make it standard
